wgsl/index.bs

```diff
@@ -6290,7 +6290,7 @@ See [[#sync-builtin-functions]].
 <table class='data'>
   <caption>Binary arithmetic expressions with mixed scalar and vector operands</caption>
   <thead>
-    <th>Preconditions<th>Conclusions<th>Semantics
+    <tr><th>Preconditions<th>Conclusions<th>Semantics</tr>
```

Comment:
nit: bikeshed allows omitting the `</tr>` closing tag, and that's what we do everywhere else, so remove for consistency

Reply:
done.